rpcs3/Input/ds4_pad_handler.h

```diff
@@ -87,6 +87,7 @@ class ds4_pad_handler final : public PadHandlerBase
 		u8 smallVibrate{ 0 };
 		std::array<u8, 64> padData{};
 		u8 batteryLevel{ 0 };
+		u8 lastBatteryLevel { 0 };
```

Comment:
We generally try to use snake_case. Don't regard the older variables that still use camelCase.
```suggestion
		u8 last_battery_level{ 0 };
```